src/views/profile/bookmarks-view/components/sidebar/index.tsx

```diff
@@ -24,6 +25,7 @@ export function ProfileBookmarksSidebar() {
 								title: 'Bookmarks',
 								fullPath: '/profile/bookmarks',
 								isActive: true,
+								trailingIcon: <IconBookmark16 />,
```

Comment:
Subtask: [Add bookmark icon next to sidebar menu item](https://app.asana.com/0/0/1203075873125155/f)